org.adempiere.base/src/org/compiere/model/MAssetAddition.java

```diff
@@ -575,6 +575,10 @@ public String prepareIt()
 		
 		MAsset asset = getA_Asset(true);
 		
+		if (MAsset.A_ASSET_STATUS_New.equals(asset.getA_Asset_Status()) && !isA_CreateAsset()) {
+			throw new AssetException("Set create asset to true for new assets");

```

Comment:
if this is an invariant, should probably be enforce via callout, readonly logic and beforeSave event instead.